src/robotFactory.js

```diff
@@ -1,10 +1,108 @@
 'use strict';
 
-class BaseRobot {}
+const defaultCoords = {
+  x: 0,
+  y: 0,
+  z: 0,
+};
+
+const minStep = 1;
+
+class BaseRobot {
+  constructor(
+    name,
+    weight,
+    coords = {},
```

Comment:
You set default value for `coords` in `constructor`.
```suggestion
    coords,
```